bin/mocha

```diff
@@ -52,7 +52,9 @@ process.argv.slice(2).forEach(arg => {
       args.unshift(arg);
       break;
     default:
-      if (arg.indexOf('--harmony') === 0) {
+      if (arg.indexOf('--experimental-modules') === 0) {
+        args.unshift(arg);
```

Comment:
The only reason this line is necessary is to ensure that `--harmony` flags aren't moved to come before it, as this needs to be an early flag.

Reply:
lgtm, though this code sucks right here